packages/vite-node/package.json

```diff
@@ -80,7 +80,7 @@
     "pathe": "^0.2.0",
     "source-map": "^0.6.1",
     "source-map-support": "^0.5.21",
-    "vite": "^3.0.0"
```

Comment:
Shouldn't this be moved to `peerDependencies`?

Reply:
Hm, technically you don't need vite for `ViteNodeRunner` 🤔 

But vite-node can also be used as CLI, where it is required, so